src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL;
+    private static final Properties DB_PROPERTIES;
+    private static final String pathToConfig = "src/main/resources/configuration.properties";
```

Comment:
JAVA Code Convention violation here
```suggestion
    private static final String pathToConfig = "src/main/resources/configuration.properties";
```